### PR TITLE
Comment adverts on sensitive pages fix

### PR DESCRIPTION
**Pull request by Calanthe**

### CODE REVIEW

**Comment by sndrs** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/commercial/comment-adverts.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ define([
             !$adSlotContainer.length ||
             !config.switches.discussion ||
             !identityApi.isUserLoggedIn() ||
+            (config.page.section === 'childrens-books-site' || config.page.shouldHideAdverts) ||
```
Comment (gklopper) [inline static/src/javascripts/projects/common/modules/commercial/comment-adverts.js]: This not the same as 2 separate `or` statements?

`a || (b || c) || d == a || b || c || d`

**Comment by gklopper** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/commercial/comment-adverts.js (hunk 1 above)
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/comment-adverts.js]: Probably it is, but this is how we usually test sensitive pages (those two values together). I think it is more visually appealing to leave it as it is now.

**Pull request merged**